packages/cardpay-cli/scheduled-payment/estimate-execution.ts

```diff
@@ -0,0 +1,120 @@
+import { Argv } from 'yargs';
+import { getSDK } from '@cardstack/cardpay-sdk';
+import { getEthereumClients, getConnectionType, NETWORK_OPTION_ANY } from '../utils';
+import { Arguments, CommandModule } from 'yargs';
+
+export default {
+  command:
+    'estimate-execution <moduleAddress> <tokenAddress> <amount> <payeeAddress> <fixedUSDFee> <percentageFee> <maxGasPrice> <gasTokenAddress> <salt> <payAtOrRecursDayOfMonth> <gasPrice>',
+  describe: 'Enable scheduled payment module on the safe',
+  builder(yargs: Argv) {
+    return yargs
+      .positional('moduleAddress', {
+        type: 'string',
+        description: 'The address of scheduled payment module',
+      })
+      .positional('tokenAddress', {
+        type: 'string',
+        description: 'The address of the token being transferred',
+      })
+      .positional('amount', {
+        type: 'string',
+        description: 'Amount of tokens you would like transferred (in the smallest units of token)',
+      })
+      .positional('payeeAddress', {
+        type: 'string',
+        description: 'The address of the payee of scheduled payment',
+      })
+      .positional('fixedUSDFee', {
+        type: 'number',
+        description: 'Fixed USD fee (e.g. 0.25)',
+      })
+      .positional('percentageFee', {
+        type: 'number',
+        description: 'Percentage fee (e.g. 5%, 0.05)',
+      })
+      .positional('percentageFee', {
+        type: 'number',
+        description: 'Percentage fee (e.g. 5%, 0.05)',
+      })
+      .positional('maxGasPrice', {
+        type: 'string',
+        description: 'maximum gas price (in the smallest units of gas token)',
+      })
+      .positional('gasTokenAddress', {
+        type: 'string',
+        description: 'The address of the gas token',
+      })
+      .positional('salt', {
+        type: 'string',
+        description: 'Arbitrary string to make SP unique',
+      })
+      .positional('payAtOrRecursDayOfMonth', {
```

Comment:
Can we have two different args for these fields? Since they can have quite different values I feel it's better to separate them

Reply:
updated